release-notes/v0.11.0.md

```diff
@@ -0,0 +1,55 @@
+I am excited to announce the release of godog v0.11.0-rc1.
+
+Here follows a summary of Notable Changes, the Non Backward Compatible Changes and Deprecation Notices.
+The full change log is available [here](https://github.com/cucumber/godog/blob/master/CHANGELOG.md#v0110-rc1).
+
+
+Notable Changes
+---------------
+
+### Write test report to file
+godog is now able to write the report to a file.
+
+- `--format cucumber` will continue to write the report to `stdout`
+
+- `--format cucumber:report.json` will instead write the report to a file named `report.json`
+
+### Upload artifacts to the github release
+The releases on github now include prebuilt binaries for:
+- Linux for amd64 and arm64
+- MacOs (Darwin) for amd64
+
+### Restructure of the codebase with internal packages
+A lot of the internal code that used to be in the main godog package has been moved to internal packages.
+
+The reason for this is mainly for decoupling to allow for simpler tests and to make the codebase easier to work with in general.
+
+Non Backward Compatible Changes
+-------------------------------
+
+### Concurrency Formatter
+`ConcurrencyFormatter` is now removed since the deprecation in `v0.10.0`.
+
+### Run and RunWithOptions
+`Run` and `RunWithOptions` are now removed since the deprecation in `v0.10.0`.
```

Comment:
Any replacements for it?

Reply:
yea, shown in the `v0.10.0` release-docs, since that is where they became deprecated
maybe link to that document or do you think we should write it out here as well?

Reply:
Of course, it says it in the text, missed that 👍 No need to write it out again, maybe a link? Up to you.